trinity/plugins/registry.py

```diff
@@ -28,4 +31,5 @@ def is_ipython_available() -> bool:
     AttachPlugin() if is_ipython_available() else AttachPlugin(use_ipython=False),
     FixUncleanShutdownPlugin(),
     TxPlugin(),
+    EthstatsPlugin(),
```

Comment:
Alphabetical order would likely be better. Here and when importing the plugin above.

Reply:
Fixed, changes will visible be when I update branch.